sdk/resources/azure-mgmt-resource/tests/test_mgmt_resource_private_link.py

```diff
@@ -0,0 +1,22 @@
+﻿# coding: utf-8
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+import azure.mgmt.resource
+import azure.mgmt.resource.privatelinks
+from devtools_testutils import AzureMgmtRecordedTestCase, RandomNameResourceGroupPreparer, recorded_by_proxy
+
+class TestMgmtResourceLinks(AzureMgmtRecordedTestCase):
+
+    def setup_method(self, method):
+        self.client = self.create_mgmt_client(
+            azure.mgmt.resource.privatelinks.ResourcePrivateLinkClient
+        )
+
+    @RandomNameResourceGroupPreparer()
+    @recorded_by_proxy
+    def test_private_link_list(self):
+        result = list(self.client.resource_management_private_link.list().value)
```

Comment:
why call `.value` here?

Reply:
What returns by the list() method here is not an iterable object https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/resources/azure-mgmt-resource/azure/mgmt/resource/privatelinks/v2020_05_01/models/_models_py3.py#L136

Reply:
Got it then I think we don't need `list (...)` here.

Reply:
got it, updated